src/applications/mhv/secure-messaging/components/MessageActionButtons.jsx

```diff
@@ -25,10 +27,20 @@ const MessageActionButtons = props => {
       type: Actions.Message.SET_THREAD_PRINT_OPTION,
       payload: printOption,
     });

```

Comment:
I believe you were attempting to cleanup printOption. Do we still need this argument `printOption` and the `dispatch` block?